### PR TITLE
Not yet

### PR DESCRIPTION
**Pull request by charizard8888**
… if there's no issue with that :p

Merge this later

### CODE REVIEW

**Comment by charizard8888** [comment]
not yet

**Comment by charizard8888** [comment]
Damn error

**Comment by charizard8888** [comment]
Reseolved it but didn't mark it as resolved

**Comment by Spandamn** [comment]
Fix conflicts

**Comment by charizard8888** [comment]
FIxed I think

File: config/formats.js [hunk 1]
```diff
@@ -423,6 +423,8 @@ exports.Formats = [
 			let name = toId(pokemon.name);
 			//let opp = toId(source.name);
 			if (name === 'spandan') {
+				this.add('raw|<div class="broadcast-red"><h2>The server has crashed! Reason:</h2>File Spandan.exe could not be located</div>');
```
Comment (Spandamn) [inline config/formats.js]: Kek. 8/8 m8

Comment (charizard8888) [inline config/formats.js]: That's the best part :]

**Comment by Spandamn** [comment]
No isnt fixed

**Comment by charizard8888** [comment]
Where is the conflict I can't find it :s

**Comment by Spandamn** [comment]
Neither me
@XpRienzo halp

**Comment by charizard8888** [comment]
Most probably in the CSS hmmface

**Comment by XpRienzo** [comment]
Wait, I'll fix and merge.

**Comment by charizard8888** [comment]
:]

**Comment by charizard8888** [comment]
Thanks

**Pull request merged**